packages/teleterm/src/mainProcess/runtimeSettings.ts

```diff
@@ -79,28 +82,25 @@ function getTshHomeDir() {
   return tshPath;
 }
 
-function getTshBinaryPath() {
+// binDir is used in the packaged version to add tsh to PATH.
+// tshBinPath is used by Connect to call tsh directly.
+function getBinaryPaths(): { binDir?: string; tshBinPath: string } {
```

Comment:
Nit: `getBinariesPaths()`?

Reply:
I've always been pluralizing only the second word but I've never actually checked if that's correct, do you have any resources about that?

If it those were paths to files and not binaries I'd probably call it `getFilePaths` and not `getFilesPaths` so I think `getBinaryPaths` is correct here. 🤔

Reply:
Ok, your argumentation seems reasonable, let's keep it as it is :)